common/app/common/NewNavigation.scala

```diff
@@ -382,21 +375,25 @@ object NewNavigation {
       SectionsLink("au/lifeandstyle/health-and-wellbeing", healthAu, Life)
     )
 
-    def getSectionLinks(sectionName: String, edition: Edition): Seq[NavLink] = {
+    def getSectionLinks(sectionName: String, edition: Edition): Tuple2[Seq[NavLink], Seq[NavLink]] = {
       val sectionList = sectionLinks.filter { item =>
         item.pageId == sectionName
       }
 
       if (sectionList.isEmpty) {
-        News.getPopularEditionalisedNavLinks(edition).drop(1)
+        val mostPopular = News.getEditionalisedSubSectionLinks(edition).mostPopular.drop(1)
+        val leastPopular = News.getEditionalisedSubSectionLinks(edition).leastPopular
+
+        (mostPopular, leastPopular)
       } else {
         val section = sectionList.head
-        val parentSection = section.parentSection.getPopularEditionalisedNavLinks(edition).drop(1)
+        val mostPopular = section.parentSection.getEditionalisedSubSectionLinks(edition).mostPopular.drop(1)
+        val leastPopular = section.parentSection.getEditionalisedSubSectionLinks(edition).leastPopular
 
-        if (parentSection.contains(section.navLink) || NewNavigation.PrimaryLinks.contains(section.navLink)) {
-          parentSection
+        if (mostPopular.contains(section.navLink) || NewNavigation.PrimaryLinks.contains(section.navLink)) {
+          (mostPopular, leastPopular)
         } else {
-          Seq(section.navLink) ++ parentSection
+          (Seq(section.navLink) ++ mostPopular, leastPopular.filter(_.title != section.navLink.title))
```

Comment:
I think there is a more functional way to write this method that avoids as many `if...else`.

```
sectionLinks.filter(_.pageId == sectionName)
  .headOption
  .map { section =>
    ...
  }
  .getOrElse {
    ...
  }
```

and actually I would go farther than that and move the `...` (and subparts of those) into separate methods.  I think it would help readability to reduce the length of this method and give names to help explain certain parts.  E.g. `(Seq(section.navLink) ++ mostPopular, leastPopular.filter(_.title != section.navLink.title))`